e2etests/tests/bgp_session_status.go

```diff
@@ -408,3 +409,21 @@ func statusFormatFor(ip string) string {
 func ipFor(statusIP string) string {
 	return strings.ReplaceAll(statusIP, "-", ":")
 }
+
+// Ensures that all the frr-k8s daemons run a clean FRR config.
+// TODO: this is needed for a CI flake, remove once we move to FRR 10.x
+func ensureCleaned(cl client.Client) {
+	Eventually(func() error {
```

Comment:
Ideally you could add `  GinkgoHelper()` in the  function

From https://onsi.github.io/ginkgo/#mental-model-how-ginkgo-handles-failure

Any function in which GinkgoHelper() is called is tracked by Ginkgo and ignored when a failure location is being computed. This allows you to build reusable test helpers and trust that the location presented to the user will always be in the spec that called the helper, and not the helper itself.

Reply:
done (although I'm not familiar with how this affects the output)

Reply:
If fails, ginkgo will tell you it failed line 76, and not line 417